.gitignore

```diff
@@ -6,4 +6,4 @@
 .DS_Store
 *.out
 build/
-linux_build.sh
+linux_build.sh
```

Comment:
nitpicking here, but try to always leave a blank line at the end of a file, to avoid unnecessary diff in the future

Reply:
Thanks! I had forgotten to add automatic endlines on vscode 😅